### DISCUSSION TITLE
External server projects only provisioned default traefik certificate

### DISCUSSION THREAD

**Discussion by johngrimsey** [Dokploy/dokploy#504] [Q&A] [answered]
I have a two server Dokploy setup which works great, apart from I seem to be having trouble with SSL on anything hosted on my secondary instance.

On my primary Dokploy server I have successfully configured a domain with SSL.

On my second server,I have tried a static nixpacks build, as well as Logto (docker compose). The second server's traefik config seems ok and the acme.json file is not empty. But I always get ERR_CERT_AUTHORITY_INVALID error when trying to access the domains.

I have tried reloading traefik on both instances, and have always ensured that DNS is correct before adding domains.

Am I doing something obviously wrong, or could it be a bug? Thanks!

**Comment by johngrimsey** [comment]
@Siumauricio is this me doing something wrong do you think? Grateful for any pointers you can give 🙏

**Reply by johngrimsey** [reply]
`docker container ls` on the secondary server is showing `PORTS` as `80/tcp` for the container in question.

The domain is setup via the primary server's Dokploy, setting HTTPS to true and Container Port to 80. The container itself is a static Angular app, using the nixpack default settings.

**Reply by Siumauricio** [reply, marked as answer]
Hi, I have some questions

1. what dns provider are you using? cloudflare? if so what mode are you using flexible or strict?
2. Did you point the `A` record to the remote server?
3. can you also share how did you configured your domain

**Comment by johngrimsey** [comment]
Thanks @Siumauricio you've cleared up a misunderstanding on my part. Pointing the A record to the secondary server and recreating the domain solved it. Thanks for being so responsive!
